src/VisualStudio/Core/Def/TaskList/VisualStudioTaskListService.cs

```diff
@@ -83,13 +86,13 @@ private async Task StartAsync(Workspace workspace)
                 await workspaceStatus.WaitUntilFullyLoadedAsync(_threadingContext.DisposalToken).ConfigureAwait(false);
 
                 // Wait until the task list is actually visible so that we don't perform pointless work analyzing files
-                // when the user would not even see the results.  When we actually do register the analyer (in
-                // _listener.Start below), solution-crawler will reanalyze everything with this analayzer, so it will
+                // when the user would not even see the results.  When we actually do register the analyzer (in
+                // _listener.Start below), solution-crawler will reanalyze everything with this analyzer, so it will
                 // still find and present all the relevant items to the user.
                 await WaitUntilTaskListActivatedAsync().ConfigureAwait(false);
 
-                // Now that we've started, let the VS todo list know to start listening to us
-                _eventListenerTracker.EnsureEventListener(_workspace, this);
+                // Now that we've started, create the actual VS todo list and have them hookup to us.
+                _ = new VisualStudioTaskListTable(workspace, _threadingContext, _tableManagerProvider, this);
```

Comment:
we had two layers of IEventListener indirection.  One to make this service type.  The next to go fire and have the task list window code actually start up and start populating.  Just removed both indirections.